hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/RSRpcServices.java

```diff
@@ -320,6 +321,10 @@
    */
   private static final boolean DEFAULT_REJECT_BATCH_ROWS_OVER_THRESHOLD = false;
 
+  public static final String CLIENT_BOOTSTRAP_NODE_COUNT = "hbase.client.bootstrap.node.count";
```

Comment:
nit: "...node.limit" conveys the intent better? (meaning we use limit or less than limit if fewer are available).

Reply:
Let me change.